rules/otp.bzl

```diff
@@ -203,6 +203,7 @@ def _otp_alert_digest_impl(ctx):
         inputs = inputs,
         arguments = [args],
         executable = tc.tools.opentitantool,
+        use_default_shell_env = True,
```

Comment:
Could you add a comment here as to why this is needed?

Reply:
Done. Thanks